bluesky_widgets/models/run_engine_client.py

```diff
@@ -58,6 +61,13 @@ def __init__(self, zmq_server_address=None, user_name="GUI Client", user_group="
         #   using positions is consistent. Empty list - no items are selected
         self._selected_history_item_pos = []
 
+        # Current directory (used for opening/saving files).
+        self.current_dir = None
```

Comment:
Should this set to `os.cwd()` by default?

Reply:
It is set to `os.cwd()` later. I think general flow of data and setting should be revisited at certain point to take into account the most of the possible use cases.